dist/src/radon.js

```diff
@@ -281,7 +281,13 @@ var Script = /** @class */ (function () {
         script.reduce(function (acc, item) {
             var op = new Operator(cache, _this.scriptId, acc, item, _this.onChildrenEvent());
             _this.operators.push(op);
-            return op.operatorInfo.outputType;
+            // if the outputType is same return its input type to be able to calculate the options
```

Comment:
```suggestion
            // If the `outputType` is `same` (a pseudo-type), return the input type so the available methods can be guessed correctly.
```

Reply:
Yet this left me wondering... does `same` exist anymore?

Reply:
The output type for ArrayFilter and ArraySome operators is `same`

Reply:
I agree that `ArrayFilter` must be `same` or `Array<inner>`, but shouldn't `ArraySome` outptut a `RadonBoolean`?

Reply:
Anyway, you resolved by doubt :+1: